db/video/vfr/README.txt

```diff
@@ -1,4 +1,4 @@
 # Transcode file to VFR, add keyframes:
 ffmpeg -i input.mp4 -vsync vfr -vf setpts='N/(25*TB)' -g 10 -keyint_min 10  output.mp4
 
-Files in this directory were created based on files in cfr directory using command above.
+Files in this directory were created based on files in cfr directory combining command above with commands used for transcoding.
```

Comment:
Running command above on HEVC file makes it H264. We need to combine changing frame rate with transcoding in single FFMPEG run.